docs/index.html

```diff
@@ -0,0 +1,2007 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ecosystem website</title>
```

Comment:
```suggestion
    <title>Qiskit Ecosystem</title>
```
https://github.com/qiskit-community/ecosystem/issues/569 tracks adding a description. We need to write copy (the wording) for it, which I don't want to block this PR on.